lib/index.ts

```diff
@@ -47,18 +44,20 @@ const model = <T extends Document>(name: string, schema: Schema | SchemaDefiniti
 			returnObject[key] = model[key];
 		}
 	});
-	return returnObject as any;
+	return returnObject;
 };
 model.defaults = {
 	...require("./Model/defaults").custom
 };
 
-export = {
+export  {
```

Comment:
Would be great if the linter could also enforce not having 2 spaces between `export` and `{` haha. Lots of linter wishes here...

```suggestion
export {
```